src/frontend/web_application/src/components/ContactAvatarLetter/index.jsx

```diff
@@ -29,10 +29,12 @@ const ContactAvatarLetter = ({ contact, size, isRound }) => {
 
   const letterClassNameSize = classNameSize ? `${classNameSize}__letter` : null;
 
+  const letterClassName = className ? `${className}__letter` : null;
```

Comment:
I think we should not do that, it is too much flexible and a bit magic; 
I mean someone can set the classname `the-very-specific-interface` and "magically" he can make a style `.the-very-specific-interface__letter`. 

Then if we change how  `${className}__letter` is used, (we remove it or we change DOM structure or ...) it might breaks the design and we will not able to see it.